### ISSUE TITLE
Summary of errors in form has conflicting roles

### ISSUE THREAD

**Issue by niekderksen** [incluud/accessible-astro-components#128]
In the Forms component - summary of errors:

1. I assume a typo in `arialive="polite"`, there is a missing hyphen. It should be `aria-live="polite"`
2. Now there are also conflicting roles; I see `role="none"` and `aria-live="polite`

Here the role="none" indicates the element has no semantic meaning.

---

Solution:
Use `role="alert"` for immediate error notifications or role="status" for less urgent statusmessages. `role="status/alert"` have implied `aria-live`-values

```
<div class="notification" role="alert">
```
